elevate-survey/controllers/survey.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * name : controllers/project.js
+ * author : Adithya Dinesh
+ * Date : 22-Aug-2024
+ * Description : Orchestration controller for Samiksha service
+ */
+
+const routeConfigs = require('../constants/routes')
+const requesters = require('../utils/requester')
+const common = require('../constants/common')
+
+/**
+ * Fetch resources from Samiksha service.
+ * @name fetchSamikshaTemplates
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} responses  
+ * @returns {JSON} - List of resources
+ */
+/**
+ * req.body = {
+ * 		organization_id : 1,
```

Comment:
@Prajwal17Tunerlabs please remove un wanted commented lines

Reply:
@rakeshSgr Akash Asked @adithyadinesh0412  to add this